### PR TITLE
🏃‍♂️‍➡️Adds runbooks and ways of working and update indexes

### PR DESCRIPTION
**Pull request by bagg3rs**
# Pull Request Objective

Migrates runbooks and ways of working from `analytical-platform-runbooks` repo into mono'ish repo for better search and one less repo

relates to #4713 and #1569

### CODE REVIEW

File: source/documentation/joining-our-team/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## General Information
+
+- Our team have a stand up everyday at 10:00
+- We plan our sprints using GitHub Projects.
+- We have our team Retrospective every 2 weeks.
```
Comment (julialawrence) [inline source/documentation/joining-our-team/index.html.md.erb]: 3 weeks

Comment (julialawrence) [reply source/documentation/joining-our-team/index.html.md.erb]: worth noting our sprints are 3 weeks too. :)

File: source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+owner_slack: "#analytical-platform-notifications"
+title: Working in Modernisation Platform
+last_reviewed_on: 2024-07-18
+review_in: 1 months
+---
+
+# <%= current_page.data.title %>
+
+The Analytical Platform makes use of the Modernisation Platform as a hosting solution for some of it's services (e.g. [analytical-platform-ingestion](https://github.com/ministryofjustice/modernisation-platform-environments/tree/main/terraform/environments/analytical-platform-ingestion)). As such, it is important to understand how to develop solutions effectively in this context.
```
Comment (julialawrence) [inline source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb]:
```suggestion
The Analytical Platform makes use of the Modernisation Platform as a hosting solution for some of its services (e.g. [analytical-platform-ingestion](https://github.com/ministryofjustice/modernisation-platform-environments/tree/main/terraform/environments/analytical-platform-ingestion)). As such, it is important to understand how to develop solutions effectively in this context.
```

File: source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Running Terraform in Modernisation Platform
+
+> In this case we will use the [analytical-platform-ingestion](https://github.com/ministryofjustice/modernisation-platform-environments/tree/main/terraform/environments/analytical-platform-ingestion) environment.
+
+1. Clone the [modernisation-platform-environments](https://github.com/ministryofjustice/modernisation-platform-environments) repository.
```
Comment (julialawrence) [inline source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb]:
```suggestion
1. Clone the [modernisation-platform-environments](https://github.com/ministryofjustice/modernisation-platform-environments) repository and open it in Visual Studio Code.
```

File: source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+1. Start the Dev Container. `Cmd + Shift + P` -> `Dev Containers: Reopen in Container`
+1. Navigate to the environment you are operating in - `cd terraform/environments/analytical-platform-ingestion`.
+1. To run a `terraform init` + `terraform plan` run the following command:
```
Comment (julialawrence) [inline source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb:20]: I can't quite tell, are these all being rendered as `1.`s?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+- **Collaboration:** Foster a culture of open communication and collective problem-solving.
+- **Quality:** Uphold high standards in our work, ensuring reliability and efficiency.
+- **Innovation:** Embrace new ideas and technologies that propel us and our platforms forward.
+- **Transparency:** Maintain clear visibility into our processes, decisions, and progress.
+
+## Team Roles
+
+- **Team Members:** Comprising a mix of developers, data, platform and software engineers, responsible for the delivery of tasks and active participation in all Agile ceremonies.
+- **Product Manager (PM):** Oversees the product backlog, ensuring it aligns with stakeholder needs and business objectives.
+- **Delivery Manager (DM):** Facilitates the Agile process, removing blockers and ensuring timelines are met.
```
Comment (julialawrence) [inline source/documentation/ways-of-working/index.html.md.erb:21]: Technical Architect?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+### Schedule
+
+This team uses **3 week Sprints** This is our timeframe for planning, executing, and reviewing work.
+
+The team has the following ceremonies:
+
+| Ceremony                                      | Occurence | Day             | Time          |
+| --------------------------------------------- | --------- | --------------- | ------------- |
+| [Stand Up](#stand-up)                         | Daily     | Monday - Friday | 10:00 - 10:15 |
+| [Backlog Management](#backlog-management)     | Weekly    | Thursday        | 14:00 - 15:00 |
```
Comment (julialawrence) [inline source/documentation/ways-of-working/index.html.md.erb]: We have a backlog management session?

File: source/documentation/ways-of-working/index.html.md.erb [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+| [Refinement](#refinement)                     | Weekly    | Tuesday         | 11:00 - 12:00 |
+| [Sprint Planning](#sprint-planning)           | Every third week | Thursday        | 10:00 - 11:00 |
+| [Sprint Review](#sprint-review)               | Every third week | Thursday        | 10:00 - 11:00 |
+| [Sprint Retrospective](#sprint-retrospective) | Every third week | Wednesday       | 10:00 - 11:00 |
+| [Show & Tell](#show-and-tell)                 | Every third week | Thursday        | 15:00 - 15:30 |
```
Comment (julialawrence) [inline source/documentation/ways-of-working/index.html.md.erb]: Wednesday now.

**Review by julialawrence — CHANGES_REQUESTED** [review]

File: source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb (hunk 4 above)
Comment (bagg3rs) [reply source/documentation/runbooks/002-working-in-modernisation-platform.html.md.erb:20]: no markdown sorts them out.

File: source/documentation/ways-of-working/index.html.md.erb (hunk 6 above)
Comment (bagg3rs) [reply source/documentation/ways-of-working/index.html.md.erb]: I just lifted and shifted, they did need a review!

File: source/documentation/ways-of-working/index.html.md.erb (hunk 5 above)
Comment (bagg3rs) [reply source/documentation/ways-of-working/index.html.md.erb:21]: I don't do anything apparently

Comment (julialawrence) [reply source/documentation/ways-of-working/index.html.md.erb:21]: I am not falling into such an obvious TARP.

**Review by julialawrence — APPROVED** [review]

**Pull request merged**